formula-android/src/main/java/com/instacart/formula/fragment/FormulaFragment.kt

```diff
@@ -5,57 +5,53 @@ import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
 import androidx.fragment.app.Fragment
-import com.instacart.formula.RenderView
+import com.instacart.formula.Cancelable
+import com.instacart.formula.android.FeatureView
+import com.instacart.formula.android.ViewFactory
 import com.jakewharton.rxrelay3.BehaviorRelay
-import io.reactivex.rxjava3.disposables.Disposable
 
-class FormulaFragment<RenderModel> : Fragment(), BaseFormulaFragment<RenderModel> {
+class FormulaFragment : Fragment(), BaseFormulaFragment<Any> {
```

Comment:
Why not to keep the narrowed `RenderModel` generic type?

Reply:
We don't have any information about the generic value since we are receiving a `FragmentKey` which doesn't contain `RenderModel` type info. This actually brings a good point that it might make sense to clean up `FormulaFragment` and `BaseFormulaFragment` API surface further.